robowflex_movegroup/include/robowflex_movegroup/services.h

```diff
@@ -72,7 +72,7 @@ namespace robowflex
              *  \param[in] path Path to execute.
              *  \return True on success, false on failure.
              */
-            bool executeTrajectory(const robot_trajectory::RobotTrajectory &path);
+            bool executeTrajectory(robot_trajectory::RobotTrajectory &path);
```

Comment:
I'd like to keep the interface such that `path` is `const`. For adding the time parameterization we should just make a copy and parameterize that, and spit out a warning for the user.